templates/concept.html

```diff
@@ -74,7 +75,7 @@
 						<input type="hidden" value="http://www.ft.com/ontology/concept/Concept" name="directType">
 					{{/if}}
 					<button
-						class="o-buttons {{#if prioritised}}o-buttons--primary{{/if}} o-buttons-icon o-buttons-icon--icon-only myft-ui__button--pin"
+						class="myft-ui__pin-button"
```

Comment:
Not sure if we should use the namespace as the block level. So maybe something more like `myft-pin-button`